specification/ml/_types/TrainedModel.ts

```diff
@@ -71,7 +71,10 @@ export class TrainedModelDeploymentStats {
   inference_count: integer
   /** The unique identifier for the trained model. */
   model_id: Id
-  /** The deployent stats for each node that currently has the model allocated. */
+  /**
+   * The deployment stats for each node that currently has the model allocated.
+   * @availability stack
```

Comment:
There are useful stats in this object so of which could be aggregated and lifted up to the top level as we have done with inference_count below.

We will want to review what model stats are presented in serverless, I'll open an issue for that

Reply:
Thanks.

The ES team is doing something similar for enrich stats - see https://github.com/elastic/elasticsearch/pull/97472.